web/src/components/snapshots/ConfigureSnapshots.jsx

```diff
@@ -26,6 +27,38 @@ class ConfigureSnapshots extends React.Component {
     });
   };
 
+  isVelero10OrNewer = () => {
+    const { snapshotSettings } = this.props;
+    if (!semverjs.valid(snapshotSettings?.veleroVersion)) {
+      return true;
+    }
+
+    const velero10Semver = semverjs.coerce("1.10");
+    const actualVeleroSemver = semverjs.coerce(snapshotSettings?.veleroVersion);
+
+    return semverjs.gte(actualVeleroSemver, velero10Semver);
+  };
+
+  getFSBackupComponentName = () => {
+    const newName = "Node Agent";
+    const oldName = "Restic";
+
+    if (this.isVelero10OrNewer()) {
+      return newName;
+    }
+    return oldName;
+  };
+
+  getFSBackupComponentFlags = () => {
+    const newFlags = ["--use-node-agent", "--uploader-type=restic"];
+    const oldFlags = ["--use-restic"];
+
+    if (this.isVelero10OrNewer()) {
+      return newFlags;
+    }
+    return oldFlags;
+  };
+
```

Comment:
```suggestion
  getFSBackupComponentFlags = () => 
    this.isVelero10OrNewer() ? 
      ["--use-node-agent", "--uploader-type=restic"] : 
      ["--use-restic"];
```

can tighten this up.